docs/how-to/create-simple-chat-app.md

```diff
@@ -105,10 +103,14 @@ ipfs = await Ipfs.create({
 
 #### Setup
 
-Please note that this how-to uses example star nodes — however, those won't necessarily always be accessible. Currently it's important to either find a reliable star node or host your own. You can host your own by following the instructions for a [native setup](https://github.com/libp2p/js-libp2p-webrtc-star%23rendezvous-server-aka-signaling-server) or for a [Docker container](https://github.com/libp2p/js-libp2p-webrtc-star/blob/master/DEPLOYMENT.md) which includes Nginx (for SSL). If you opt for the native setup, we cover the Nginx reverse proxy process and SSL cert retrieval later in this post.
+Please note that this how-to uses example star nodes — however, those won't necessarily always be accessible. Currently it's important to either find a reliable star node or host your own. You can host your own by following the instructions for a [native setup](https://github.com/libp2p/js-libp2p-webrtc-star/tree/master/packages/webrtc-star-signalling-server) or for a [Docker container](https://github.com/libp2p/js-libp2p-webrtc-star/blob/master/packages/webrtc-star-signalling-server/DEPLOYMENT.md) which includes Nginx (for SSL). If you opt for the native setup, we cover the Nginx reverse proxy process and SSL cert retrieval later in this post.
 
 ### p2p-circuit
 
+:::warning
+This section is currently only relevant for go-ipfs versions **before** v0.11.0 as it's about circuit relay v1. There is currently no solution available to directly replace it, though most of the work [has been completed](https://github.com/libp2p/go-libp2p-relay-daemon).
```

Comment:
@TheDiscordian was there a hard blocker that made you write this? iiuc configuring it to be `RelayV1: true`  should be drop-in replacement, no?

```suggestion
This section is currently only relevant for go-ipfs versions **before** v0.11.0 as circuit relay v1 server is no longer provided with go-ipfs. There is currently a standalone solution available  [go-libp2p-relay-daemon](https://github.com/libp2p/go-libp2p-relay-daemon#readme) configured as `RelayV1: true`.
```